### PR TITLE
[py] PEP 484 type hints for selenium.webdriver.remote.utils

### PR DESCRIPTION
**Pull request by hoefling**
### Description

This PR adds type hints to `selenium.webdriver.remote.utils`.

### Motivation and Context

This is a follow-up of #9482 to address #9480.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: py/selenium/webdriver/remote/utils.py [hunk 1]
```diff
@@ -20,24 +20,31 @@
...
+    Path = Union[str, os.PathLike[str]]
+else:
+    Path = Union[str, os.PathLike]
 
 
 LOGGER = logging.getLogger(__name__)
 
 
-def format_json(json_struct):
+def format_json(json_struct: Any) -> str:
```
Comment (hoefling) [inline py/selenium/webdriver/remote/utils.py]: @AutomatedTester looks like this function is unused.

File: py/selenium/webdriver/remote/utils.py [hunk 2]
```diff
@@ -20,24 +20,31 @@
...
     return json.dumps(json_struct)
 
 
-def load_json(s):
+def load_json(s: Union[str, bytes]) -> Any:
     return json.loads(s)
 
 
-def unzip_to_temp_dir(zip_file_name):
+def unzip_to_temp_dir(zip_file_name: Path) -> Optional[str]:
```
Comment (hoefling) [inline py/selenium/webdriver/remote/utils.py]: @AutomatedTester this function seems to be unused as well.

**Comment by AutomatedTester** [comment]
If there are methods that are not used then let's delete them.

**Comment by hoefling** [comment]
@AutomatedTester done, please review again!

**Review by AutomatedTester — APPROVED** [review]

**Pull request merged**